facebook/OpenBIC
# fby35: gl: Configure sensors following the hardware design

Summary:
- Create sensnor table and sdr table.
- Implement plat_hook.c to handle sensor's read method.
- Sensor table and sdr table following the sensor table version 20230111.

Test Plan:
- Check Sensor number, name and threshold in BMC console : Pass
- Check Sensor number and name in BIC console : Pass

Test Log:
BMC console :
    root@bmc-oob:~# sensor-util slot3
    slot3:
    MB_INLET_TEMP_C    (0x1) :   22.00 C     | (ok) | UCR: NA | UNC: NA | UNR: 149.86 | LCR: NA | LNC: NA | LNR: NA
    MB_OUTLET_TEMP_C   (0x2) :   27.00 C     | (ok) | UCR: 28.32 | UNC: NA | UNR: 149.86 | LCR: NA | LNC: NA | LNR: NA
    FIO_FRONT_TEMP_C   (0x3) :   22.00 C     | (ok) | UCR: NA | UNC: NA | UNR: NA | LCR: NA | LNC: NA | LNR: NA
    ```

BIC console :
    uart:~$ platform sensor list_all
    ---------------------------------------------------------------------------------
    [0x1 ] MB_INLET_TEMP_C            : tmp75
    [0x2 ] MB_OUTLET_TEMP_C           : tmp75
    [0x3 ] FIO_FRONT_TEMP_C           : tmp75
    ```
    [0x35] MB_VR_VCCINF_PWR_W         : xdpe15284
    [0x36] MB_VR_VCCD0_PWR_W          : xdpe15284
    [0x37] MB_VR_VCCD1_PWR_W          : xdpe15284
    ---------------------------------------------------------------------------------
